backend/tracim_backend/views/core_api/system_controller.py

```diff
@@ -153,6 +154,15 @@ def user_custom_properties_schema(self, context, request: TracimRequest, hapic_d
         )
         return {"json_schema": custom_properties_api.get_json_schema()}
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG_SYSTEM_ENDPOINTS])
+    @check_right(is_user)
+    @hapic.output_body(PreFilledAgendaEventSchema())
+    def pre_filled_agenda_event(self, context, request: TracimRequest, hapic_data=None):
```

Comment:
I don't think system controller is the correct place to put this, this is agenda app related.

Reply:
fixed